### PR TITLE
feat(util): add nunjucks template support for file path

### PR DESCRIPTION
**Pull request by albertlieyingadrian**
Context https://github.com/promptfoo/promptfoo/issues/1685

### CODE REVIEW

File: src/util/index.ts [hunk 1]
```diff
@@ -1152,7 +1152,10 @@ export function maybeLoadFromExternalFile(filePath: string | object | Function |
     return filePath;
   }
 
-  const finalPath = path.resolve(cliState.basePath || '', filePath.slice('file://'.length));
+  // Render the file path using Nunjucks
+  const renderedFilePath = nunjucks.renderString(filePath, { env: process.env });
```
Comment (mldangelo) [inline src/util/index.ts]: We already add this here: https://github.com/promptfoo/promptfoo/blob/main/src/util/templates.ts#L30 

```suggestion
  const renderedFilePath = nunjucks.renderString(filePath);
```

**Review by mldangelo — COMMENTED** [review]
Thanks for the quick fix @albertlieyingadrian! Would you mind adding one test case to https://github.com/promptfoo/promptfoo/blob/main/test/util.test.ts#L48 ?

**Comment by albertlieyingadrian** [comment]
@mldangelo Added a unit test and applied your suggestion

File: src/util/index.ts (hunk 1 above)
Comment (albertlieyingadrian) [reply src/util/index.ts]: @mldangelo I tried removing this and it didn't  pass the unit test https://github.com/promptfoo/promptfoo/actions/runs/10890208463/job/30218459298

Comment (albertlieyingadrian) [reply src/util/index.ts]: nvm. Decided to use global nunjucks engine instead https://github.com/promptfoo/promptfoo/pull/1688/commits/f94359e10cfad8e330e2bd9a25121811fa0efb0e

Comment (albertlieyingadrian) [reply src/util/index.ts]: @mldangelo Decided to use global nunjucks engine instead to centralize the env loading

```javascript
const renderedFilePath = getNunjucksEngine().renderString(filePath, {});
```

**Comment by mldangelo** [comment]
This is perfect! Thank you @albertlieyingadrian!

**Comment by albertlieyingadrian** [comment]
Thanks! @mldangelo when is this going to be released?

**Comment by mldangelo** [comment]
Tonight!

**Pull request merged**